### PR TITLE
kernelci.org: add blog about UX Analysis project

### PR DESCRIPTION
**Pull request by gctucker**
Add a blog post about the UX Analysis project starting now with ProFUSION following the RFP published earlier in 2023.

### CODE REVIEW

**Comment by gctucker** [comment]
Now on the staging website: https://static.staging.kernelci.org/blog/posts/2023/ux-analysis/

File: kernelci.org/content/en/blog/posts/2023/ux-analysis/index.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+The UX Analysis project is broken down into three milestones with four weeks
+assigned to each, so a total of twelve weeks.  Some breaks will also be
+scheduled during this time so the calendar end date is likely to be in March
+2024.
+
+### M1: Result of interviews
+
+Some interviews will be carried out with various stakeholders to help refine
+user stories and start modeling the user experience and some design aspects.
```
Comment (patersonc) [inline kernelci.org/content/en/blog/posts/2023/ux-analysis/index.md:31]:
```suggestion
user stories and start modelling the user experience and some design aspects.
```

However, perhaps `modeling` is an American spelling, so technically may be correct ;)

**Comment by padovan** [comment]
Looks good to me too!

**Comment by gctucker** [comment]
OK the PR hasn't been actually approved but I'll assume it's ready to be merged now based on the earlier comments.

**Pull request merged**